tests/virt_control_plane_test.go

```diff
@@ -28,6 +28,7 @@ import (
 	v1 "k8s.io/api/apps/v1"
 	k8sv1 "k8s.io/api/core/v1"
 	"k8s.io/api/policy/v1beta1"
+	"k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Please alias this to something like `k8serrors`. The current name is masking the std lib [errors](https://pkg.go.dev/errors).

Reply:
It is a pretty standard convention in our repo. Would you mind tackling this in separate PR?

Reply:
We have a convention to mask std golang libraries? If it is done, I hope we can agree it is wrong.
You are introducing it now here, so I hope we can start avoiding this now and not add it to the list of fixes.

Is there a specific reason you prefer this convention?

Reply:
I don't prefer it. By convention, I meant it seems to be a pretty common thing.

Reply:
Fixed.